src/collections/_documentation/accounts/quotas/index.md

```diff
@@ -0,0 +1,212 @@
+    level="warning"
+%}
+
+*Filter by Issue.* When you are unable to take immediate action on an issue, but it continues to occur, Sentry supports deleting and discarding that issue from the issue details page. Navigate to **[Project] » Project Settings » Inbound Filters**, then click “delete and discard future events.” This setting deletes most data associated with the issue and filters out matching events before they count against your quota.
+
+{% include components/alert.html
+    title="Note"
+    content="Deleting and discarding issues is available only for Business and Enterprise plans"
+    level="warning"
+%}
+
+## Spike Protection
+
+A spike is both a **large** and **temporary** increase in event volume. Sentry's spike protection prevents huge overages from consuming your event capacity. We use your historical event volume to implement a dynamic rate limit which discards events when you hit its threshold. This dynamic rate limit is designed to protect you from short-term spikes. The threshold will increase if your increased volume persists for many hours.
+
+When spike protection is activated, we limit the number of events accepted in any minute to:
+
+`maximum(20, 6 x average events per minute over the last 24 hours)`
+
+_The 24 hour window ends at the beginning of the current hour, not at the current minute._
+
+What this means is that if you experience a spike, we will temporarily protect you, but if the increase in volume is sustained, the spike protection limit will gradually increase until Sentry finally accepts all events.
+
+### How Does Spike Protection Help? {#how-does-spike-protection-help}
+
+Because Sentry bills on monthly event volume, spikes can consume your Sentry capacity for the rest of the month. If it really is a spike, spike protection drops events during the spike to try and conserve your capacity. We also send an email notification to the Owner when spike protection is activated.
+
+## Attribute Limits
+
+Sentry imposes hard limits on various components within an event. While the limits may change over time and vary among attributes, most individual attributes are capped at 512 characters. Additionally, certain attributes also limit the maximum number of items. Commonly-trimmed values might include, for example:
+
+-     Stack frame variables. Stack traces with large frame counts will be trimmed (the middle frames are dropped). We limit to a maximum of 250 frames. Only 50 are retained unaltered (modulo usual trimming), with the remaining 200 removing `vars`, `pre_context` and `post_context`.
+-     `extra` data is limited to approximately 256k characters, and each item is capped at approximately 16k characters.
+-     Breadcrumbs and their values
+-     Generic attributes like the event’s label also have limits, but are more flexible depending on their case. For example,  messages and message parameters are limited to 1024 characters.
```

Comment:
This is a bit verbose since we're already listing examples.

```suggestion
-     Error messages and message parameters are limited to 1024 characters.
```

Reply:
Thank you.